ftl-build/x86_64-musl/Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG ALPINE_VER=3.14
-FROM alpine:${ALPINE_VER}
+ARG ALPINE_VER
+FROM amd64/alpine:${ALPINE_VER:-v3.14}
```

Comment:
Why change this from `VAR=value` to variable expansion?

Reply:
I remembered seeing on the Docker official site that it was better to use that syntax, but I was wrong
It isn't said anywhere